src/components/buttons/CopyButton/_private/CopyButtonIcon.tsx

```diff
@@ -17,8 +19,16 @@ interface ICopyButtonIconProps {
 
 export const CopyButtonIcon = (props: ICopyButtonIconProps) => {
 	const { variant } = props;
-	const { checkmark, checkmarkBackground } = CopiedStateIconVariants;
-	const { link } = UncopiedStateIconVariants;
+	const { checkmark, checkmarkBackground, checkmarkStroke } = CopiedStateIconVariants;
+	const { link, copy } = UncopiedStateIconVariants;
+
+	if (variant === copy) {
+		return (
+			<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+				<path fill-rule="evenodd" clip-rule="evenodd" d="M9 6C9.55228 6 10 6.44772 10 7V12V13C10 13.5523 9.55228 14 9 14H3C2.44772 14 2 13.5523 2 13V7C2 6.44772 2.44772 6 3 6H4H9ZM4 4V3C4 1.34315 5.34315 0 7 0H13C14.6569 0 16 1.34315 16 3V9C16 10.6569 14.6569 12 13 12H12V13C12 14.6569 10.6569 16 9 16H3C1.34315 16 0 14.6569 0 13V7C0 5.34315 1.34315 4 3 4H4ZM6 4V3C6 2.44772 6.44772 2 7 2H13C13.5523 2 14 2.44772 14 3V9C14 9.55228 13.5523 10 13 10H12V7C12 5.34315 10.6569 4 9 4H6Z" fill="#51BB7B"/>
```

Comment:
I think `fill-rule` & `clip-rule` need to be camelcase, I'm seeing some errors in the console about it.

Reply:
Aha good catch, I just assumed that error wasn't me for some reason 🥴